src/main/java/de/tudarmstadt/lt/wsi/WSI.java

```diff
@@ -231,11 +216,20 @@ public static void main(String args[]) throws IOException {
 		int N = Integer.parseInt(cl.getOptionValue("N"));
 		int n = Integer.parseInt(cl.getOptionValue("n"));
 
-
 		StringIndexGraphWrapper<Float> graphWrapper = GraphReader.readABCIndexed(inReader, false, N, minEdgeWeight);
 		WSI cwd = null;
 		if (cl.getOptionValue("clustering").toLowerCase().equals("cw")) {
 			cwd = new WSI(graphWrapper, n, ClusteringAlgorithm.ChineseWhispers);
+			String cwOption = cl.getOptionValue("cwOption");
+			CW.Option option;
+			if ("DIST_LOG".equals(cwOption)) {
+				option = Option.DIST_LOG;
+			} else if ("DIST_NOLOG".equals(cwOption)) {
+				option = Option.DIST_NOLOG;
+			} else {
```

Comment:
Maybe we should print a help message and exit if none of the three options matches the provided parameter